py/setup.py

```diff
@@ -41,6 +41,8 @@
     ],
     "doc": ["pydoc-markdown"],
     "openai-agents": ["openai-agents"],
+    # These should only be installed for linting import errors, not for tests.
+    "lint": ["anthropic"],
```

Comment:
i might be reading this wrong but do you still need this for linting if you've removed the import and type annotations below?

Reply:
@choochootrain it's actually for linting the applications in `py/examples`. I could omit them from that and remove any mention of it in imports. 🤷‍♂️